package.json

```
@@ -86,6 +86,7 @@
         "uglify-js": "2.8.29",
         "ui-select": "npm:ui-select#0.13.2",
         "underscore": "1.8.3",
+        "url-polyfill": "^1.1.10",
```

@jc00ke our current convention is to use concrete versions in our package file rather than auto-updating to new dependencies

```suggestion
        "url-polyfill": "1.1.10",
```